src/utils/debug/texture-selector-utils.ts

```diff
@@ -42,14 +63,29 @@ export function selectDebugTextureForTile(tile, uvDebugTexture) {
       tile.userData.originalTexture =
         material.pbrMetallicRoughness.baseColorTexture.texture.source.image;
     }
-    material.pbrMetallicRoughness.baseColorTexture.texture.source.image =
-      uvDebugTexture;
-    tile.content.material = { ...tile.content.material };
+    const width =
+      material.pbrMetallicRoughness.baseColorTexture.texture.source.image.width;
+    const height =
+      material.pbrMetallicRoughness.baseColorTexture.texture.source.image
+        .height;
+    cropTexture(uvDebugTexture, width, height).then((uvDebugTexture) => {
```

Comment:
Make `selectDebugTextureForTile` also async. skipping tests is not a way

Reply:
Done